nifi-nar-bundles/nifi-influxdb-bundle/nifi-influxdb-processors/src/main/java/org/apache/nifi/processors/influxdb/AbstractInfluxDBProcessor.java

```diff
@@ -56,7 +56,8 @@
             .build();
 
     public static final PropertyDescriptor INFLUX_DB_CONNECTION_TIMEOUT = new PropertyDescriptor.Builder()
-            .name("InfluxDB Max Connection Time Out (seconds)")
+            .name("influxdb-max-connection-time-out")
```

Comment:
Unfortunately I think we don't want this change to happen. As soon as a processor is released, changing the ``.name()`` property would be a breaking change for existing flows with this processor. I think we might want to revisit all the processor for a 2.0 release but it'd need to be discussed and we would need to provide tools for a proper migration.

Reply:
Agreed, please keep the `name` the same but update `displayName`, that should give you what you want without breaking existing processors.

Reply:
i'm going to merge and just remove the name line while keeping displayName